packages/wrangler/src/config/index.ts

```diff
@@ -158,498 +152,20 @@ export function readPagesConfig(
 	};
 }
 
-export const readRawConfig = (configPath: string | undefined): RawConfig => {
+export const experimental_readRawConfig = (
```

Comment:
You need to re-export this on the entry file (`src/cli.ts`) to expose the API

Reply:
🤦 8aecc4ea08ac72924d33d9de91bd331b96b07a46